docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+</p>
+
+4. The **View Recovery Seed Phase** page displays your secret recovery phrase. Write it down and keep it safe. Click **Close**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-account-seedphrase.png" alt="Cobalt SeedPhrase" width="300" height="200" />
+</p>
+
+#### Lock 
+The Cobalt wallet allows you to lock and unlock your wallet. Use the lock option for security reasons. 
+
+1. In the **Account** menu, click **Lock**. Your wallet is locked. The **Welcome Back!** page appears.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-account-welcomeback.png" alt="Cobalt WelcomeBack" width="300" height="200" />
+</p>
+
+2. To unlock, enter your password.  Click **Unlock**.
+
+#### Connected Sites
+
+The Cobalt wallet displays the network addresses that are connected. The Connected Sites page is populated with all site URLs connected to your account, automatically. You can disconnect to a network address if desired. 
+
+1. In the **Account** menu, click **Connected Sites**. A list of connected network addresses is displayed.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-connect-site.png" alt="Cobalt ConnectSite" width="300" height="200" />
+</p>
+
+2. If desired, you can disconnect from a site by hovering your cursor over the chain icon, where a **Disconnect button** appears. Click the **Disconnect icon** to delete the address.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-connectsite-disconnect.png" alt="Cobalt DisConnectSite" width="300" height="200" />
+</p>
+
+#### Manage Account
+
+You can manage your wallet accounts. The Manage Account option allows you to rename your account or hide your account from being displayed in Cobalt. This option also allows you to add a new account.
+
+1. In the **Account** menu, click **Manage Account**. A list of valid wallet names and associated addresses are displayed.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-manageacct-list.png" alt="Cobalt ManageAcct List" width="300" height="200" />
+</p>
 
+2. Click the **pencil icon** next to the wallet name you want to modify. The Rename page appears. In the Account name field, make the changes to the wallet  name. Click **Confirm**.
 
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-manageacct-rename.png" alt="Cobalt ManageAcct Rename" width="300" height="200" />
+</p>
+
+3. Toggle the **eye icon** next to the wallet name you want to be visible or not. This option does not delete the account but rather hides the account in the setting option.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-manageacct-visible.png" alt="Cobalt ManageAcct Visible" width="300" height="200" />
+</p>
+
+4. You can also add an account to your wallet. In the **Manage Account** page, click **Add account**.
+
+   **Note:** When adding a new account, the account’s seed phrase is the same as its initial phrase.
+
+<p>
+<img src="/img/docs/get-started/gobie-manageacct-newacct.png" alt="Cobalt ManageAcct NewAcct" width="300" height="200" />

```

Comment:
There is a tiny "g" at the top right corner of the image. Can w crop it?

Reply:
replaced screenshot with a new one